docs/book/language-reference.md

```diff
@@ -100,7 +100,13 @@ The input `string` is a JSON Web Token encoded with JWS Compact Serialization. J
 | <span class="opa-keep-it-together">``time.parse_rfc3339_ns(value, output)``</span> | 1 | ``output`` is ``number`` representing the time ``value`` in nanoseconds since epoch.`` |
 
 > Multiple calls to the `time.now_ns` built-in function within a single policy
-evaluation query will always the same value.
+evaluation query will always return the same value.
+
+### <a name="graphs"/>Graphs
+
+| Built-in | Inputs | Description |
+| --- | --- | --- |
+| <span class="opa-keep-it-together">``walk(x, [path, value])``</span> | 0 | ``path`` is ``array`` representing a pointer to ``value`` in ``x``.  Queries can use ``walk`` to traverse documents nested under ``x`` (recursively). |
```

Comment:
I was a bit confused reading this the first time, I think partly because the output value is a 2-place array.  I couldn't tell if `walk` returned the list of all <path, value> pairs or a single pair.  Maybe clarify that ``walk`` is a relation here?  

Also, is the signature ``walk(x, [path, value])`` or ``walk(x, path, value)`` clearer?